storage/file_persister_test.go

```diff
@@ -74,3 +79,164 @@ func TestLocalFilePersister(t *testing.T) {
 		})
 	}
 }
+
+func TestRemoteFilePersister(t *testing.T) {
+	t.Parallel()
+
+	basePath := "screenshots"
+
+	tests := []struct {
+		name                    string
+		path                    string
+		data                    string
```

Comment:
`data` is purposefully generic since the persister can upload any type of data, not only screenshots. I think it's clear based on the fact that it's data within a test that is testing some sort of persisting behaviour.

Reply:
I still think a comment like this can be there for clarity.

Reply:
Resolved in ef9e44df2a5352b9f5ccf94d3dbb218bcbdb1f83